argocd/overlays/moc-infra/applications/envs/moc/infra/operate-first/opf-vault.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: opf-vault
+spec:
+  project: operate-first
+  source:
+    repoURL: https://github.com/operate-first/apps.git
+    targetRevision: HEAD
+    path: vault/overlays/moc/infra
+  destination:
+    name: infra
```

```suggestion
    name: moc-infra
```
`destination.name ` should be `moc-infra` (see [example](https://argocd-server-argocd.apps.moc-infra.massopen.cloud/applications/acme-operator-infra?resource=&node=argoproj.io%2FApplication%2Fargocd%2Facme-operator-infra%2F0&tab=manifest))

---

see cluster list [here](https://argocd.operate-first.cloud/settings/clusters) for reference